.gitignore

```diff
@@ -15,7 +15,6 @@ awscreds.env
 *.out
 
 # Kubernetes Generated files - skip generated files, except for vendored files
-zz_generated.*.go
 !vendor/**/zz_generated.*
```

Comment:
Line 19 is no longer needed as well

Reply:
We don't use any vendored modules, so this is not necessary. Thank you